lib/models/ObjectMD.js

```diff
@@ -1028,6 +1030,33 @@ class ObjectMD {
         return this;
     }
 
+    /**
+     * Create or update the microVersionId field
+     *
+     * This field can be used to force an update in MongoDB (in case
+     * no other metadata field changes) or to manage conflicts during
+     * concurrent updates, using conditions on the microVersionId
+     * field.
```

Comment:
Would be interesting to mention that it is also useful to detect a change when fields change but object version does not change (e.g. when ingesting a putObjectTagging coming from S3C to Zenko).

Reply:
... And bring clean active-active semantics for S3 calls that are not clean (like putObjectTagging for instance).

Reply:
done